pkgs/tools/wayland/cliphist/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "cliphist";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "sentriz";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-CZW7dhOd7E74VNjnvhxvSSUZQtbkGi4uRUM9YQCuJZw=";
+  };
+
+  vendorSha256 = "sha256-rxE4fNS3THWAfqIpKLPQ+a9dl1fDnIlMfvblb7JqfBc=";
```

Comment:
Can you double-check this vendor hash?

I get:
```
hash mismatch in fixed-output derivation '/nix/store/99jsd1iagz05g5wrybrpydagwbw2y3pw-cliphist-0.1.0-go-modules':
  wanted: sha256:05vwdar6zrgngr68k763aybmvbzrs2rjhad2gs07ak5psiy3h4dg
  got:    sha256:1qq37jbjcw4w91g13w7xa1f337r5v8xpxzs6qbih26r4zc795cjj
```

Reply:
Hi, I am new to Nix and this is my first submission - still very much of a neophyte. The package installs fine when I do `nix-env -f . -iA cliphist` from the local repo. Can you please tell me how should I go about double-checking it?

Reply:
Derivations that are passed hashes are so called [fixed-output derivations](https://nixos.org/manual/nix/stable/#fixed-output-drvs). If you pass it a hash of a derivation you already have in your store¹, it will be used instead of getting the new one, ignoring that the URL changed.

You can double-check by changing the first letter of the hash to `0` or `1`, which will hopefully not exist in your store and force Nix to re-download it and tell you the correct hash.

---

1. Their names also have to match, which is rarer with Go modules where it depends on `name` (`${name}-go-modules`) but can happen.